src/Chapter14/Listing14.08.UsingThePlusAndMinusDelegateOperators.cs

```diff
@@ -3,11 +3,13 @@
     using System;
     using Listing14_01;
     using Listing14_05;
-
+    #region INCLUDE
```

Comment:
Would it be simpler to just move the #region INCLUDE to just surround the inner section instead of the #exclude right next to an include?